layers/tls.go

```diff
@@ -0,0 +1,205 @@
+// Copyright 2012 Google, Inc. All rights reserved.
+// Copyright 2009-2011 Andreas Krennmair. All rights reserved.
+// TLS decoder is a contribution from Jose Selvi
+//
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+package layers
+
+import (
+	"encoding/binary"
+	"errors"
+
+	"github.com/google/gopacket"
+)
+
+// TLSType defines the type of data after the TLS Record
+type TLSType uint8
+
+// TLSType known values.
+const (
+	TLSchange_cipher_spec TLSType = 20
```

Comment:
Use CamelCase everywhere, and capitalize the first letter after the acronym.

TLSChangeCipherSpec
TLSAlert
TLSHandshake
...